config/webpack.common.js

```diff
@@ -56,14 +55,19 @@ module.exports = {
       },
     ],
   },
-  plugins: [
-    new CleanWebpackPlugin([C.BUNDLE_PROJECT_DIR], {root: C.ROOT_DIR}),
-    new webpack.optimize.CommonsChunkPlugin({
-      name: 'vendors',
-      minChunks: function(module) {
-        return module.context && module.context.indexOf('node_modules') !== -1;
+  optimization: {
+    splitChunks: {
```

Comment:
This is from https://webpack.js.org/plugins/split-chunks-plugin/#split-chunks-example-2 and I think it's equivalent to the original config.